src/main/java/com/gradle/enterprise/conventions/customvalueprovider/LocalBuildCustomValueProvider.java

```diff
@@ -21,16 +23,28 @@ public boolean isEnabled() {
     @Override
     public void accept(Settings settings, BuildScanExtension buildScan) {
         buildScan.tag("LOCAL");
-        if (Stream.of("idea.registered", "idea.active", "idea.paths.selector").anyMatch(it -> getConventions().systemPropertyProvider(it).isPresent())) {
+        if (isRunningInIdea()) {
```

Comment:
🤔 Does this cause a CC miss when switching between IDEA and non-IDEA invocations?

Reply:
> 🤔 Does this cause a CC miss when switching between IDEA and non-IDEA invocations?

Yes, but IDEA usually injects its init scripts, so you won't get hit anyway. Nevertheless, we can read these in background too, though because of the ValueSource-in-background issue it is a lot of ceremony to keep this compliant with pre-7.6. Do we actually need to support the legacy Gradle versions?

Reply:
I don't think we need to support pre-7.6 versions. Those should use an old plugin version instead.

Reply:
> I don't think we need to support pre-7.6 versions. Those should use an old plugin version instead.

Let's start with #23 then